netman/adapters/switches/cached.py

```diff
@@ -254,29 +255,25 @@ def set_interface_description(self, interface_id, description):
         # No cache to update
         self.real_switch.set_interface_description(interface_id, description)
 
-    def remove_interface_description(self, interface_id):
+    def unset_interface_description(self, interface_id):
         # No cache to update
-        self.real_switch.remove_interface_description(interface_id)
+        self.real_switch.unset_interface_description(interface_id)
 
     def set_bond_description(self, bond_number, description):
         # No cache to update
         self.real_switch.set_bond_description(bond_number, description)
 
-    def remove_bond_description(self, bond_number):
+    def unset_bond_description(self, bond_number):
         # No cache to update
-        self.real_switch.remove_bond_description(bond_number)
+        self.real_switch.unset_bond_description(bond_number)
 
-    def edit_interface_spanning_tree(self, interface_id, edge=None):
+    def set_interface_spanning_tree_state(self, interface_id, edge=None):
```

Comment:
This one is a tricky part, it's not a real boolean... it's more like a sub resource... as in

set_interface_spanning_tree_edge_state... but at the same time it does not do exactly this it does more than this...

this function is somewhat controversal in its use... i suggest you remove this one from your refactor so that we address it later